src/utils.ts

```diff
@@ -571,7 +561,26 @@ export function appendIdfAndToolsToPath() {
     "components",
     "partition_table"
   )}`;
-  modifiedEnv.PATH = `"${IDF_ADD_PATHS_EXTRAS}${path.delimiter}${modifiedEnv.PATH}"`;
+
+  if (process.platform === "win32") {
+    modifiedEnv.Path =
+      path.join(modifiedEnv.IDF_PATH, "tools") +
+      path.delimiter +
+      modifiedEnv.Path;
+    if (modifiedEnv.Path && !modifiedEnv.Path.includes(extraPaths)) {
+      modifiedEnv.Path = extraPaths + path.delimiter + modifiedEnv.Path;
+    }
+    modifiedEnv.Path = `${IDF_ADD_PATHS_EXTRAS}${path.delimiter}${modifiedEnv.Path}`;
+  } else {
+    modifiedEnv.PATH =
```

Comment:
Sorry for a dumb question, but is it not possible to do something like
```
if (process.platform == "win32") {
  path_name = "Path";
} else {
  path_name = "PATH";
}
modifedEnv[path_name] = ...
```
to avoid duplicating the code?